include/SFML/Graphics/Texture.hpp

```diff
@@ -120,8 +120,7 @@ class SFML_GRAPHICS_API Texture : GlResource
     /// This function is a shortcut for the following code:
     /// \code
     /// sf::Image image;
-    /// image.loadFromFile(filename);
-    /// texture.loadFromImage(image, area);
+    /// image.loadFromFile(filename) && texture.loadFromImage(image, area)
```

Comment:
Something is missing, either an if statement or a semicolon. I suggest an if statement.
Same goes for other changes in this file.

Reply:
Yep, just turning them into a boolean expression is more confusing 😄

Reply:
https://github.com/SFML/SFML/blob/157feec05dbf888bfbfaabf8c4ae0cb366a7f9e8/src/SFML/Graphics/Texture.cpp#L264-L265

That confused me too. It's just a snippet from the implementation of `sf::Texture::loadFromFile`.

Reply:
Ah, I guess adding the `return` might also be okay

Reply:
Are you gonna add this?

Reply:
Yeah I suppose I'll eventually do that. This PR is just not a priority for me.